lib/game_over.rb

```diff
@@ -0,0 +1,48 @@
+class GameOver
+    def win?(board)
```

Comment:
I notice that GameOver has a concept of board which is different than a Board (class). Should it take a Board (class)? or is the Board (class) not needed and can be removed?

Reply:
Good question! My thought is that it's being passed the Board class's `@board` instance variable when it's called. I think I still want a class responsible for the housing the Board data and it's functionality, but I do need that data to be accessed by other classes and methods in the game. So I'm amicable to changing this if you think there's some feature envy happening here.